src/net/peacefulcraft/trenchpvp/gamehandle/listeners/CombatLoggerListener.java

```diff
@@ -0,0 +1,91 @@
+package net.peacefulcraft.trenchpvp.gamehandle.listeners;
+
+import java.util.HashMap;
+
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.EntityDamageEvent;
+import org.bukkit.event.entity.PlayerDeathEvent;
+
+import net.peacefulcraft.trenchpvp.TrenchPvP;
+import net.peacefulcraft.trenchpvp.gamehandle.TeamManager;
+import net.peacefulcraft.trenchpvp.gamehandle.TeamManager.PlayerWideExecutor;
+import net.peacefulcraft.trenchpvp.gamehandle.player.TrenchPlayer;
+
+public class CombatLoggerListener implements Listener {
+    
+    //private static HashMap<TrenchPlayer,CombatLogger> loggers = new HashMap<>();
+
+    private static HashMap<TrenchPlayer, EntityDamageEvent> damageMap;
```

Comment:
What is the `damageMap` used for? I see that values get `.push()`ed and `.remove()`ed from it, but not where it gets used in any checks.

Reply:
Damage map would be used for the stat logging. It contains the information regarding who damaged who, with what item, for how much damage, etc. It is unused right now.